### PR TITLE
Borrowed API POST Method

### PR DESCRIPTION
**Pull request by Sana-Ullah786**
This PR contributes the POST method for Borrowed API to the Project.

- Using **/borrowed/** as **POST** request borrowed record can be created in the Database.
Please Have a look and let me know about any changes. 👀 
**Ticket Link:**
https://app.asana.com/0/1204411152293714/1204530928804120/f

### CODE REVIEW

**Review by SohaibkhanFolio3 — COMMENTED** [review]
LGTM

**Review by SohaibkhanFolio3 — APPROVED** [review]

File: src/endpoints/borrowed/post/create_borrowed.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            detail="Copy with given ID is not available",
+        )
+    user = (
+        db.scalars(
+            select(all_models.User).where(all_models.User.id == borrowed.user_id)
+        )
+        .unique()
+        .one_or_none()
+    )
+    if user is None:
```
Comment (mustansirMuz) [inline src/endpoints/borrowed/post/create_borrowed.py:52]: I think user_id should not be asked as input. The logged in user's id should be used itself. If we ask id from user then it means one user can borrow for another user, which is wrong.

**Review by mustansirMuz — CHANGES_REQUESTED** [review]
All looks good except the one small change I pointed out.

File: src/endpoints/borrowed/post/create_borrowed.py (hunk 1 above)
Comment (Sana-Ullah786) [reply src/endpoints/borrowed/post/create_borrowed.py:52]: Resolved.

**Review by mustansirMuz — APPROVED** [review]

**Pull request merged**